pkg/virt-controller/watch/drain/disruptionbudget/disruptionbudget_test.go

```diff
@@ -109,10 +110,14 @@ var _ = Describe("Disruptionbudget", func() {
 			Expect(ok).To(BeTrue())
 			Expect(patchAction.GetName()).To(Equal("pdb-" + vmi.Name))
 			Expect(patchAction.GetPatchType()).To(Equal(types.JSONPatchType))
-
-			expectedPatch := fmt.Sprintf(`[{ "op": "replace", "path": "/spec/minAvailable", "value": 1 }, { "op": "remove", "path": "/metadata/labels/%s" }]`,
-				patch.EscapeJSONPointer(v1.MigrationNameLabel))
-			Expect(string(patchAction.GetPatch())).To(Equal(expectedPatch))
+			patches := patch.New(
+				patch.WithReplace("/spec/minAvailable", 1),
+				patch.WithRemove(fmt.Sprintf("/metadata/labels/%s", patch.EscapeJSONPointer(
+					virtv1.MigrationNameLabel))),
+			)
+			expectedPatch, err := patches.GeneratePayload()
+			Expect(err).ToNot(HaveOccurred())
```

Comment:
I am not sure if we should use production code like this to build the expected value.
I would leave this as is, in a raw string that shows what is expected.

But I will let you decide if it makes sense or not.